apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/Trace.cs

```diff
@@ -55,15 +57,15 @@ namespace Google.Devtools.AspNet
     /// 
     /// Docs: https://cloud.google.com/trace/docs/
     /// </remarks>
-    public sealed class CloudTrace
+    public sealed class Trace
```

Comment:
What's it colliding with? I didn't think we had anything called `CloudTrace` at that point. Maybe we should merge as-is, then I'll fiddle and see what I can do.